spring-cloud-aws-secrets-manager/src/main/java/io/awspring/cloud/secretsmanager/SecretsManagerPropertySource.java

```diff
@@ -77,13 +78,19 @@ public Object getProperty(String name) {
 	private void readSecretValue(GetSecretValueRequest secretValueRequest) {
 		try {
 			GetSecretValueResponse secretValueResponse = source.getSecretValue(secretValueRequest);
-			Map<String, Object> secretMap = jsonMapper.readValue(secretValueResponse.secretString(),
-					new TypeReference<Map<String, Object>>() {
-					});
+			try {
+				Map<String, Object> secretMap = jsonMapper.readValue(secretValueResponse.secretString(),
+						new TypeReference<Map<String, Object>>() {
+						});
 
-			for (Map.Entry<String, Object> secretEntry : secretMap.entrySet()) {
-				LOG.debug("Populating property retrieved from AWS Parameter Store: " + secretEntry.getKey());
-				properties.put(secretEntry.getKey(), secretEntry.getValue());
+				for (Map.Entry<String, Object> secretEntry : secretMap.entrySet()) {
+					LOG.debug("Populating property retrieved from AWS Parameter Store: " + secretEntry.getKey());
+					properties.put(secretEntry.getKey(), secretEntry.getValue());
+				}
+			} catch (JsonParseException e) {
+				// If the secret is not a JSON string, then it is a simple "plain text" string
+				LOG.debug("Populating property retrieved from AWS Parameter Store: " + secretValueResponse.name());
```

Comment:
While you are here change typo with logging to this:
`LOG.debug("Populating property retrieved from AWS Secrets Manager: " + secretValueResponse.name());`
:)